internals/daemon/daemon.go

```diff
@@ -795,6 +796,13 @@ func New(opts *Options) (*Daemon, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Propagate a valid plan (empty is also valid) to the subscribed
+	// overlord managers.
+	planMgr := ovld.PlanManager()
+	err = planMgr.Load()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Since the plan is not associated with its own manager, I've had to remove the lazy loading that happened in the previous design. We now have (and in the future will have) many managers that each may have APIs which could require them to have a ```plan```. I've decided to have a simple flow of updates from the Plan Manager to the plan change subscribers, so we need to have the Plan Manager get the plan as soon as possible, and update the consumers as soon as possible so they can function.

The extreme example is the ```SetServiceArgs``` call that happens even before the overlord loop has started. This prompted me to move this immediately after ```overlord.New()```. I know all the subscriptions are done, so I may as well proceed with loading the plan from disk.

Reply:
moved.